src/interactive/types.ts

```diff
@@ -35,6 +36,7 @@ export interface IInteractiveSensorData {
 export const defaultAuthoredState: IAuthoredState = {
   useFakeSensor: false,
   singleReads: false,
+  enablePause: true,
```

Comment:
Should this default to `true`?  I thought we wanted to default to not showing the pause button.

Reply:
I would agree that wanting the pause button would likely be rare, at least nobody has given me a specific use case where it would be helpful.